vignettes/sd_iv_rich_pkpd.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Single Dose IV - PKPD"
+author: "Vignette Author"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Single Dose IV - PKPD}
```

Comment:
Try to keep the vignette name as simple as possible since they'll have to type it exactly in `vignette(...)` maybe tie it directly to the dataset (I presume this is for `sd_iv_rich_pkpd`?